### ISSUE TITLE
Add a "squish math" command

### ISSUE THREAD

**Issue by trevorcampbell** [trevorcampbell/shortex#20]
We should add a command that reduces whitespace in a line of math (especially around symbols like `+`, `-`, `=`, which tend to be quite whitespace heavy.

Example -- not sure if this is possible, especially with the `&` inside, but ideally something like
```
\[
\squish{A &= b + c}
\]
```

**Comment by trevorcampbell** [comment]
As an example, this was prompted by me doing the following in a paper to squeeze some math into a two-column format
```
\hspace{-.3cm}\left(G\!+\!\tau I\right)^{-1}H(1\!-\!w) &= (\epsilon \|w\!-\!w^\star\|\!+\!\delta) v
```

**Comment by trevorcampbell** [comment]
Addressed in 8bfe81c
